src/examples/IDBBatchAtomicVFS.js

```diff
@@ -143,6 +143,7 @@ export class IDBBatchAtomicVFS extends VFS.Base {
   xRead(fileId, pData, iOffset) {
     return this.handleAsync(async () => {
       const file = this.#mapIdToFile.get(fileId);
+      iOffset = Number(iOffset)
```

Comment:
this should never go out of bounds of `Number` but we should range check just in case.